packages/wallet-sdk/src/provider/CoinbaseWalletProvider.ts

```diff
@@ -596,6 +597,76 @@ export class CoinbaseWalletProvider
     return res.result;
   }
 
+  /**
+   * @beta
+   * This method is currently in beta. While it is available for use, please note that it is still under testing and may undergo significant changes.
+   *
+   * @remarks
+   * IMPORTANT: Signature validation is not performed by this method. Users of this method are advised to perform their own signature validation.
+   * Common web3 frontend libraries such as ethers.js and viem provide the `verifyMessage` utility function that can be used for signature validation.
```

Comment:
doesn't seem like there's an existing method to do this.
we have one named `ethereumAddressFromSignedMessage`, but it's defined on the relay and makes request via WL...
![image](https://github.com/coinbase/coinbase-wallet-sdk/assets/966662/78e269de-4e36-459c-b1f1-883a9d1aad7f)